build/Jenkinsfile.pr

```diff
@@ -131,12 +131,12 @@ pipeline {
             }
 
             steps {
-                dir('src/github.com/mattermost/mattermost-server/build') {
+                dir('src/github.com/mattermost/mattermost-server') {
                     ansiColor('xterm') {
                         sh """
-                            docker-compose --no-ansi run --rm start_dependencies
-                            docker-compose --no-ansi ps
-                            docker-compose --no-ansi exec -T openldap bash -c 'ldapadd -x -D "cn=admin,dc=mm,dc=test,dc=com" -w mostest -f /${LDAP_DATA}-data.ldif';
+                        /usr/local/bin/docker-compose --no-ansi -f ./build/docker-compose.yml run --rm start_dependencies
+                        /usr/local/bin/docker-compose --no-ansi -f ./build/docker-compose.yml ps
+                        cat tests/${LDAP_DATA}-data.ldif | /usr/local/bin/docker-compose --no-ansi -f ./build/docker-compose.yml exec -T openldap bash -c 'ldapadd -x -D "cn=admin,dc=mm,dc=test,dc=com" -w mostest || true';
```

Comment:
Just out of curiosity, why the `|| true`? I see that it was there before in the Makefile, so it seems it's the right thing to do, but wondering why not fail?

Reply:
I /think/ it's there in the `Makefile` since `ldapadd` throws an error if the users already exist. I agree that, in this context, it could probably just fail instead, otherwise some downstream tests will fail mysteriously.